Monika-After-Story/MonikaModDev
# update base sprites

Removed the boob shine.

## Testing:
- Verify outfits where cleavage is visible (bikini/navyblue dress/sundress/etc) look fine
- Verify no bleeding colour from the splits
- Verify it looks good



###### pls and ty
